workspaces/homepage/docs/header.md

```diff
@@ -0,0 +1,269 @@
+| `pageTitle`         | none    | An optional property to override the title that is displayed in the browser tab.                                                                                 |
+
+## Local clock
+
+Starting with Dynamic Home Page plugin 1.1 (RHDH 1.5) the home page header can show the current time in the header.
+
+The local time can be disabled or configured with a `localClock` object. This object supports three properties: `label`, `format`, and `lang`. All of them are optional.
+
+To show the current time you must specific at least the `format`:
+
+```yaml
+dynamicPlugins:
+  frontend:
+    red-hat-developer-hub.backstage-plugin-dynamic-home-page:
+      dynamicRoutes:
+        - path: /
+          importName: DynamicHomePage
+          config:
+            props:
+              localClock:
+                format: time
+```
+
+![Header with local time](header-localtime-time.png)
+
+Showing the current date instead of time:
+
+```yaml
+dynamicPlugins:
+  frontend:
+    red-hat-developer-hub.backstage-plugin-dynamic-home-page:
+      dynamicRoutes:
+        - path: /
+          importName: DynamicHomePage
+          config:
+            props:
+              localClock:
+                format: date
+```
+
+![Header with local date](header-localtime-date.png)
+
+You can pick up more options like `both`, `full`, etc. The full list is available below.
+
+By default the format is based on the browser language settings. You can enforce a language setting with `lang`:
+
+```yaml
+dynamicPlugins:
+  frontend:
+    red-hat-developer-hub.backstage-plugin-dynamic-home-page:
+      dynamicRoutes:
+        - path: /
+          importName: DynamicHomePage
+          config:
+            props:
+              localClock:
+                format: full
+                lang: de
+```
+
+![Header with fill date and time](header-localtime-full-de.png)
+
+There is also an option to specific a label that is shown above the time. This is espally useful if you like to show additional times (see world clock section below).
+
+```yaml
+dynamicPlugins:
+  frontend:
+    red-hat-developer-hub.backstage-plugin-dynamic-home-page:
+      dynamicRoutes:
+        - path: /
+          importName: DynamicHomePage
+          config:
+            props:
+              localClock:
+                format: time
+                label: Local
+```
+
+![Header with local label and time](header-localtime-local-label.png)
+
+### Available props
+
+| Prop                | Default | Description                                                                                                                       |
+| ------------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `localClock.format` | `none`  | One of the options `none`, `time`, `timewithseconds`, `date`, `datewithweekday`, `both`, or `full` is currently supported.        |
```

Comment:
for config like
```
localClock:
                format: fulldd
```

It displays the time

Reply:
I added " Other options will show the `time`."